projects/plugins/boost/app/assets/src/js/stores/config.ts

```diff
@@ -22,6 +23,18 @@ export const dismissedPopOuts = {
 	},
 };
 
+// eslint-disable-next-line camelcase
+export const getStarted = () => {
+	update( store => ( {
+		...store,
+		site: {
+			...store.site,
+			getStarted: false,
+		},
+	} ) );
+	return saveGetStarted();
+};
```

Comment:
This method is a big improvement on the previous interface. Love it.

However, the name is slightly confusing. You're not getting started then, so calling `getStarted` seems confusing. See this comment in `PurchaseSuccess.svelte`, and how it contrasts the code:
```
		// Disable the getting started page
		getStarted();
```

I'd jokingly suggest `gotStarted`, but that's not clear either. How about `markGetStartedComplete` ?

Reply:
I have added your name as a prefix as requested

Reply:
Addressed in [b64d21e](https://github.com/Automattic/jetpack/pull/26666/commits/b64d21e5b5448752ad575edb2fb7c160b3f683a6)